src/Stratis.Features.SQLiteWalletRepository/Tables/HDTransactionData.cs

```diff
@@ -9,7 +9,18 @@ internal class HDTransactionData
         public int AccountIndex { get; set; }
         public int AddressType { get; set; }
         public int AddressIndex { get; set; }
+
+        /// <summary>
+        /// This is the exact bytes in the Script of {OutputTxId-OutputIndex}.
+        /// It should probably be called ScriptPubKey!
+        /// </summary>
         public string RedeemScript { get; set; }
+
+        /// <summary>
+        /// This is the derived ScriptPubKey from IScriptDestinationReader.
+        /// It should probably be called AddressScriptPubKey.
```

Comment:
Nice. This way we know the that this particular `ScriptPubKey` will refer to a single wallet address only.